### DISCUSSION TITLE
Pull Request Templates Scoped to Directory

### DISCUSSION THREAD

**Discussion by jmhwang7** [community/community#8229] [Pull Requests]
# The issue
Repos can have multiple pull request templates. Monorepos can contain different services that might have different considerations that should be thought through when making a change. There is no way for the user when going through the Github UI to specify which pull request template they would like to use OR know which pull request template should be used when.

# The idea
Scoping a pull request template to a directory would enable different teams to decide what they consider important when making a change to code they own. This would (hopefully) result in better changes with less bugs and more consistency.

**Comment by iddanh** [comment]
I would love to see this feature as well.

**Comment by hcrosse** [comment]
Seconding (thirding?)

**Comment by houssam7737** [comment]
+1 to this!! I'm surprised this doesn't exist already

**Comment by fanxia00** [comment]
up vote.

**Comment by ricardo-buquet** [comment]
up vote!

**Comment by romguima** [comment]
It could work similarly to the CODEOWNERS file where you specify which team owns which part of the repo, so you could have the same idea and either instead of the team you specify the template that should be used when opening the PR or just associate a PR template for each team.

**Comment by jendaprokes-enverus** [comment]
This would be useful especially in big monorepos

**Comment by mlegy** [comment]
Up vote
